core/tests/behavior/async_presign.rs

```diff
@@ -31,12 +31,13 @@ use crate::*;
 pub fn tests(op: &Operator, tests: &mut Vec<Trial>) {
     let cap = op.info().full_capability();
 
-    if cap.read && cap.write && cap.presign {
+    if cap.read && cap.write && cap.presign && cap.delete {
```

Comment:
This will skip all tests for presign. How about implement in this way:

```
if cap.read && cap.write && cap.presign {
	tests.extend(async_trials!(
            op,
            test_presign_write,
            test_presign_read,
            test_presign_stat
        ))
}
if cap.read && cap.write && cap.presign && cap.delete {
	tests.extend(async_trials!(
            op,
            test_presign_delete
        ))
}
```

Reply:
Oh, I see. It's `cap.delete` not `cap.presign_delete`.

I think we can add a check in `test_presign_delete` directly for `cap.presign_delete`.